even-more-fish-plugin/src/main/java/com/oheers/fish/addons/AddonManager.java

```diff
@@ -60,7 +62,13 @@ public ItemStack getItemStack(final String prefix, final String id) {
 
     public boolean registerAddon(final @NotNull Addon addon) {
         final String prefix = addon.getPrefix().toLowerCase(Locale.ROOT);
-        if (!addon.canRegister()) {
+        try {
+            if (!addon.canRegister()) {
+                this.loadingMap.put(prefix, true);
+                return false;
+            }
+        } catch (JavaVersionException | RequiredPluginException e) {
+            EvenMoreFish.getInstance().getLogger().warning("Addon " + e.getMessage());
```

Comment:
To prevent startup warnings, we can add `EvenMoreFish.debug("Addon " + e.getMessage())`
That way we can keep the warnings (in case something goes wrong, but hide them for most players/default addons)